src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/methodname/InputMethodNamePrivateMethodsInInterfaces.java

```diff
@@ -0,0 +1,24 @@
+//non-compiled with javac: Compilable with Java9
+package com.puppycrawl.tools.checkstyle.checks.naming.methodname;
+
+/*
+ * Config:
+ * applyToPrivate = false
+ */
+public interface InputMethodNamePrivateMethodsInInterfaces {
+
+    private void PrivateMethod() {} // ok
+
+    private static void PrivateMethod2() {} // ok
+
+    default void DefaultMethod() { // violation
```

Comment:
? 
Am I miss somethig ?
https://checkstyle.sourceforge.io/config_naming.html#ConstantName
it is about fields only.

Reply:
MethodNameCheck giving false-positives as well on same thing, since they have common way to detect visibility. I fixed both checks.

Reply:
Ok, my bad. Issue case focused me on one Check